Prace_domowe/Praca_domowa1/Grupa4/adrianna_grudzien/pd_1.ipynb

```diff
@@ -0,0 +1,1030 @@
+{
```

Comment:
<p>Jakiś powód do upodobania sobie tej zmiennej? To nie jest zarzut, po prostu warto to napisać</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>